docs/azure_jumpstart_hcibox/using_hcibox/_index.md

```diff
@@ -64,27 +64,27 @@ See the [product documentation](https://learn.microsoft.com/azure-stack/hci/upda
 
 ## Virtual machine management through Azure portal
 
-Azure Local supports [VM management through the Azure portal](https://learn.microsoft.com/azure-stack/hci/manage/azure-arc-enabled-virtual-machines). Open the [HCIBox VM provisioning documentation](../RB/) to get started.
+Azure Local supports [VM management through the Azure portal](https://learn.microsoft.com/azure-stack/hci/manage/azure-arc-enabled-virtual-machines). Open the [LocalBox VM provisioning documentation](../RB/) to get started.
```

Comment:
although these links are redirected and work, might be good to update to the new links 

![image](https://github.com/user-attachments/assets/38971ff9-f0fd-4484-92f0-c6f01ad5cc15)